dynamic-plugins/wrappers/backstage-community-plugin-github-issues/package.json

```diff
@@ -11,10 +11,10 @@
   },
   "backstage": {
     "role": "frontend-plugin",
-    "supported-versions": "1.27.7",
+    "supported-versions": "1.29.2",
```

Comment:
Is this version alright, regarding https://github.com/backstage/community-plugins/blob/main/workspaces/github-issues/plugins/github-issues/CHANGELOG.md#044?